react/features/screen-share/functions.js

```diff
@@ -23,3 +33,25 @@ export function isScreenAudioShared(state: Object) {
 export function isScreenAudioSupported() {
     return browser.isChrome() || (browser.isElectron() && isWindows());
 }
+
+/**
+ * Is any screen media currently being shared, audio or video.
+ *
+ * @param {Object} state - The state of the application.
+ * @returns {boolean}
+ */
+export function isScreenMediaShared(state: Object) {
+    return isScreenAudioShared(state) || isScreenVideoShared(state);
+}
+
+/**
+ * Is screen sharing currently active.
+ *
+ * @param {Object} state - The state of the application.
+ * @returns {boolean}
+ */
+export function isScreenVideoShared(state: Object) {
+    const localVideo = getLocalVideoTrack(state['features/base/tracks']);
+
+    return localVideo && localVideo.videoType === 'desktop';
```

Comment:
Please use `getVideoType()` here. We have been using the track videoType directly so far but it would be nice to use the getter for new code. Also, please use `VIDEO_TYPE.DESKTOP` instead of hardcoding it.

Reply:
Cool! Fixed.